src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+
+public class Todo extends Task{
+    protected String by;
```

Comment:
Is this field used? It might be better to remove it if its not needed to reduce unnecessary code.